Lib/multiprocessing/queues.py

```diff
@@ -319,6 +319,13 @@ def __init__(self, *, ctx):
         else:
             self._wlock = ctx.Lock()
 
+    def __del__(self):
```

Comment:
I removed __del__(), sorry this change was not directly related to this PR. It can be added later if needed (for ResourceWarning).